### DISCUSSION TITLE
Single switch to enable/disable position broacasting.

### DISCUSSION THREAD

**Discussion by Geddeon** [meshtastic/Discussions#104] [General]
Decided not to create two similar posts in Android and iOS topics to discuss that.
Recently decided not to broadcast my position and didn't find a single straightforward switch to enable/disable that. 
So that is my concern that up to this moment I'm still not sure if I'm not broadcasting my position to the mesh.

**Comment by mariotti** [comment]
Yup, this is a little bit confusing.

But it does not depend on IOS or Android. It depends on firmware or channel settings.

The easiest and up to date interface is the web one.
You  can configure position settings there, in particular per channel.
You need to go to the channels configuration and set it there.
BTW the default on the PRIMARY is to send the precise position, so change it first.

I actually highly suggest to not broadcast the position.

Then you can just disable the GPS in the settings/position. You can DISABLE it.

In the mobile version you should have something similar.

Check the position settings, and disable GPS.

Hope this helps a bit.

A+

**Comment by garthvh** [comment]
There is a single toggle for location on each channel, location is managed per channel.
